### DISCUSSION TITLE
How to do multiple successCondition/failureCondition?

### DISCUSSION THREAD

**Discussion by CRASH-Tech** [argoproj/argo-workflows#9319] [General]
I'm trying to do something like this, but it's not work:
`failureCondition: status.health.status != "Healthy" || status.sync.status != "Synced"`

**Comment by terrytangyuan** [comment]
Try `|` instead of `||`. 

Some known limitations: https://github.com/argoproj/argo-workflows/issues/2288 and https://github.com/argoproj/argo-workflows/issues/7143

**Reply by CRASH-Tech** [reply]
It's not work.

**Comment by CRASH-Tech** [comment]
This works:
`successCondition: status.sync.status == Synced, status.health.status == Healthy`
